modules/router/templates/router_include.conf.erb

```diff
@@ -33,6 +33,17 @@ location ~ ^\/[A-Z]+[A-Z\W\d]+$ {
   rewrite ^(.*)$ https://$host$uri_lowercase permanent;
 }
 
+<%- @app_specific_static_asset_routes.each do |alias_path, vhost_name| -%>
+# Static asset serving for <%= vhost_name %>
+set $upstream_<%= vhost_name.delete "-" %> https://<%= vhost_name %>.<%= @app_domain_internal %>;
```

Comment:
I'd be tempted to pull this line out into some variables, as I can see they're also used below in the block itself. Although I'm not sure how conventional it is to define variables in ERB files.

Reply:
Sure, I've done that now.